code-challenges.test.js

```diff
@@ -65,3 +167,30 @@ const accountTransactions3 = []
 // Expected output: []
 
 // b) Create the function that makes the test pass.
+// Pseudo code:
+// input: array
+// output: returns an array of the accumulating sum.
+// process:
+// create a function name: accSum
+// use for loop declear starting point with index 1 because nothing will be iterating on index 0
+// stop point will be less than array.length;
+// iteration will be i++:
+// for(let i = 1; i < array.length; i++)
+// then use .map method to declare maping using array  
+// replacing new value with the sum of the value of [current iteration index] and the value of [current iteration index - 1]
+// array.map(array => array[i] = array[i] + array[i - 1])
+// return array
+// pass test
+
+const accSum = (array) => {
+  for(let i = 1; i < array.length; i++) {
+   array.map(array => array[i] = array[i] + array[i - 1])
+  }
+  return array
```

Comment:
The way this function is currently set up, it is not returning what you want. Keep in mind that you are using `.map` inside the for loop, but the actual `array` param is not being modified.